upcloud_api/storage.py

```diff
@@ -153,6 +168,9 @@ def to_dict(self):
                 dict_labels.append(label.to_dict())
             body['labels'] = dict_labels
 
+        if hasattr(self, 'enrypted') and isinstance(self.enrypted, bool):
+            body['server']['enrypted'] = "yes" if self.enrypted else "no"
```

Comment:
"enrypted" typos?

Reply:
Thanks! :sweat_smile: Not something that the tests would have caught.